### PR TITLE
[xbmc][cleanup] Add missing vector includes

### PR DESCRIPTION
**Pull request by Paxxi**
Got somewhat tired of random build breakage because of missing header files so here's vector

### CODE REVIEW

**Comment by Paxxi** [comment]
jenkins build this please

**Comment by stefansaraev** [comment]
jenkins build this please

File: xbmc/cores/VideoPlayer/DVDCodecs/Video/VDPAU.h [hunk 1]
```diff
@@ -46,7 +46,7 @@
 #include "DVDVideoCodec.h"
 #include "DVDVideoCodecFFmpeg.h"
 #include <X11/Xlib.h>
-#include <X11/Xutil.h>
+j#include <X11/Xutil.h>
```
Comment (MartijnKaijser) [inline xbmc/cores/VideoPlayer/DVDCodecs/Video/VDPAU.h]: typo

**Comment by Paxxi** [comment]
jenkins build this please

**Pull request merged**